development_scripts/restore_latest_backup.sh

```
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# Configuration
+SERVER_HOST="vccfinal.net"
+SERVER_USER="ubuntu"
+SSH_KEY="~/site_2024.pem"
```

Shellcheck (0.10.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)